src/main.rs

```diff
@@ -41,6 +41,43 @@ fn convert_axis_angle_to_euler_xyz(rx: f64, ry: f64, rz: f64, angle: f64) -> (f6
 	return (euler_x, euler_y, euler_z);
 }
 
+/// Convert from Euler XYZ to AxisAngle.
+///
+/// All rotations are expected and returned as radians.
+///
+/// # Arguments
+///
+/// * `euler_rx` - The rotation around the X axis.
+/// * `euler_ry` - The rotation around the Y axis.
+/// * `euler_rz` - The rotation around the Z axis.
+///
+/// Known robots that use this format:
+/// - Staubli (NOTE: this example assumes radians values, but this robot uses degrees)
+fn convert_euler_xyz_to_axis_angle(euler_rx: f64, euler_ry: f64, euler_rz: f64) -> (f64, f64, f64, f64) {
+
+	let cx = cos(euler_rx/2.0);
+	let cy = cos(euler_ry/2.0);
+	let cz = cos(euler_rz/2.0);
+	let sx = sin(euler_rx/2.0);
+	let sy = sin(euler_ry/2.0);
+	let sz = sin(euler_rz/2.0);
+
+	// Compute the quaternion
+	let qw = -sx*sy*sz + cx*cy*cz;
```

Comment:
```suggestion
	let qw = -sx * sy * sz + cx * cy * cz;
```

Similarly here. If you want to make the grouping clearer, we can add parenthesis around the two groups?